packages/react-components/theme-designer/src/components/ExportButton/ExportButton.tsx

```diff
@@ -1,23 +1,8 @@
 import * as React from 'react';
 import { makeStyles } from '@griffel/react';
-import {
-  BrandVariants,
-  Menu,
-  MenuTrigger,
-  MenuButton,
-  MenuPopover,
-  MenuList,
-  MenuItem,
-  Theme,
-} from '@fluentui/react-components';
+import { Menu, MenuTrigger, MenuButton, MenuPopover, MenuList, MenuItem } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { Menu, MenuButton, MenuItem, MenuList, MenuPopover, MenuTrigger} from '@fluentui/react-components';
```